common/app/templates/inlineJS/blocking/analytics.scala.js

```diff
@@ -130,6 +130,38 @@ try {
         s.prop20    = tpA[2] + ':' + tpA[1];
         s.eVar20    = 'D=c20';
 
+        try {
+            var participationsKey = 'gu.ab.participations';
+            var participations = window.localStorage.getItem(participationsKey);
+
+            var abTestsParticipations = makeOmnitureABTag(participations);
+
+            // This is set globally so we can check if the use ab test participations change once the ab test runs.
```

Comment:
does this need to be global? can this be checked before AB tests run by AB tests modules?